### ISSUE TITLE
tabbing to navigate the UI

### ISSUE THREAD

**Issue by bcdaurelle** [musescore/MuseScore#33018]
### Your idea

Using the Tab key should move the user's focus/cursor around the visible elements of the UI in a simpler and more universal manner, as in a web browsers and in most modern apps. Some functions that are currently assigned to the arrow keys should be reassigned to Tab.

### Problem to be solved

Musescore 4 has an awkward/unintuitive flow for keyboard tabbing. If I click a number box in the Properties window to change the X dimension of an element, I expect that pressing the Tab key will move the cursor to the neighboring number box, for the Y dimension. Instead, tabbing rotates you through the major sections of the entire window; on to the score itself, to the upper and lower menus, etc.

Even worse, there is a lot of confusion between how navigating with Tab and arrow keys functions. For example, if I have Tabbed around so that a sub-section of the Properties menu is highlighted, I can then click [↓] to move down through the individual elements within this section. However, when I reach a number box, the same key immediately does something different: it changes the number itself instead of navigating. Clicking [→] while the numbers in a box are highlighted allows the cursor to around the numbers as text, but if the number is not highlighted (i.e. after clicking through a few times on the up/down arrows to change a number), the side arrows will now move you between number boxes. This creates a very crazy-making situation in which you cannot use the side arrows to move one direction and then the other in sequence! 

I know this is probably a big thing to change, and perhaps Musescore chose this design on purpose, but in my mind there is no question that it should behave more like browsers and other apps almost universally do, with Tab and Shift+Tab moving you around the exposed elements of the UI in a left-to-right/top-to-bottom order. And, apologies if this has been discussed elsewhere, I can't find it bc search yields a lot of results for Tableture but not 'the tab key'!

### Checklist

- [x] This request follows the <a href="https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues" target="_blank">guidelines</a> for reporting issues
- [x] I have verified that this feature request has not been logged before, by searching the <a href="https://github.com/musescore/MuseScore/issues" target="_blank">issue tracker</a> for similar requests

**Comment by MarcSabatella** [comment]
The heirarchical navigation model F6 for top level navigation, tab for groups within that, cursor keys for individual controls within that) is a newer developing design standard that has been championed by a number of accessibility experts.  Probably more apps will moving to it in coming years because it is considerably more efficient for complex windows.

But no doubt, there is room for further tweaking on terms of the specifics of where and how this model is used.  It's already requested elsewhere that the score information panel of the new score wizard be reverted to the older standard, and I definitely support that.  Probably there are other specific windows where this could be considered, but I wouldn't want to see the overall usability and efficiency of the app be compromised by just always going back to the old style.

**Comment by jmwmusic** [comment]
I think this issue is very well described in this feature request.  I do not believe that the standard that Marc has mentioned applies in any way to dialog boxes or many of the other panels in MuseScore that behave more like dialogs.  The hierarchical system users encounter in other applications use F6 to get to areas such as footers and side bars that are not being used for the general input and editing and other elements remain tabable.  Feedback from users on the MuseScore Accessibility groups.io group is unanimous that users find this current system confusing, has stopped some learning the software and even seasoned users fall again and again into missing the existence of elements or having to back track because they automatically hit tab to move to the next item.  I don't know where the standard mentioned comes from, but do wonder if it was created with little input from actual screen reader users and as explained here by the author also creates all sorts of issues with combo boxes, spin boxes, etc.

There are other open tickets regarding these issues:  See #20056, #16739, #14343, #27498, #28513, #20515, #20516, #25807,  

There are more issues that can't be fixed till the tabbing is or navigation would be lost.
